adagrams/game.py

```diff
@@ -1,11 +1,114 @@
+import random
+
+letter_bank = {
+"A" : 9,	
+"B" : 2,
+"C" : 2,	
+"D" : 4,	
+"E" : 12,	
+"F" : 2,	
+"G" : 3,	
+"H" : 2,	
+"I" : 9,
+"J" : 1,	
+"K" : 1,	
+"L" : 4,
+"M" : 2,
+"O": 8,
+"P": 2,
+"Q": 1,
+"R": 6,
+"S": 4,
+"T": 6,
+"U": 4,
+"V": 2,
+"W": 2,
+"X": 1,
+"Y": 2,
+"Z": 1,
+}
+
 def draw_letters():
-    pass
+    
+    letters = []
+    
+    for letter, count in letter_bank.items():
+       
+        letters.extend([letter] * count)
+    random.shuffle(letters)
+    hand = letters[:10]
+    return hand
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+   
+    word = word.upper()
+    
+    letter_bank_copy = list(letter_bank)
+    
+    for letter in word:
+        if letter not in letter_bank_copy:
+            return False
+        letter_bank_copy.remove(letter)
+
+    return True
+
+
+def score_word(word): 
+    
+    score_dict = {
+"A" : 1,	
+"B" : 3,
+"C" : 3,	
+"D" : 2,	
+"E" : 1,	
+"F" : 4,	
+"G" : 2,	
+"H" : 4,	
+"I" : 1,
+"J" : 8,	
+"K" : 5,	
+"L" : 1,
+"M" : 3,
+"O": 1,
+"P": 3,
+"Q": 10,
+"R": 1,
+"S": 1,
+"T": 1,
+"U": 1,
+"V": 4,
+"W": 4,
+"X": 8,
+"Y": 4,
+"Z": 10,
+}
```

Comment:
[nit] Please fix this formatting and indentation! Your coworkers will be happier if you fix it before pushing it up :)

Reply:
Oh this one, I was curious if there is a cleaner way to do this? It's so long. Also seems like I may have shifted some indentations when I saved and pushed. Whoops, thank you for catching.

Reply:
I think highlighting the lines and `tab`ing or `shift + tab`ing to the correct format should work (it should have more indentation than `score_dict`), but there might be a VSCode extension for Python that autoformats on save... If you remind me later we can google together or something